var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -59,7 +59,7 @@ def url_for_version(self, version):
     depends_on('mpi', when='+mpi')
     depends_on('fftw', when='+ksapce')
```

Comment:
This was not introduced by your PR but would you mind correcting this to `+kspace`?

Reply:
Done!